### PR TITLE
trying to get this merged

### PR DESCRIPTION
**Pull request by cameronaaron**

### CODE REVIEW

File: portal.go [hunk 1]
```diff
@@ -453,12 +453,7 @@ func (portal *Portal) markRead(intent *appservice.IntentAPI, eventID id.EventID,
 		return nil
 	}
 
-	// Only send the read update if it is more recent than the last read time
-	if portal.ReadAt.After(readAt) {
-		return nil
-	} else {
-		portal.ReadAt = readAt
-	}
+	portal.ReadAt = readAt
```
Comment (tulir) [inline portal.go:456]: the ReadAt field should be removed entirely

Comment (cameronaaron) [reply portal.go]: just removed it in the next commit

Comment (cameronaaron) [reply portal.go]: Thank you

Comment (cameronaaron) [reply portal.go]: Now?

**Comment by cameronaaron** [comment]
closes #195

File: portal.go [hunk 2]
```diff
@@ -2159,7 +2146,7 @@ func (portal *Portal) HandleiMessage(msg *imessage.Message) id.EventID {
 			}
 
 			overrideSuccess = true
-		} else if msg.IsEdited {
+		} else if msg.IsEdited && dbMessage.Part == 0 {
```
Comment (trek-boldly-go) [inline portal.go:2149]: Tulir said dbMessage.Part > 0 not ==

File: portal.go [hunk 3]
```diff
@@ -2143,11 +2135,6 @@ func (portal *Portal) HandleiMessage(msg *imessage.Message) id.EventID {
 	// If the message exists in the database, handle edits or retractions
 	if dbMessage != nil && dbMessage.MXID != "" {
 
-		// Clean up the text so the length is accurate
-		msg.Text = strings.ReplaceAll(msg.Text, "\ufffc", "")
-		msg.Subject = strings.ReplaceAll(msg.Subject, "\ufffc", "")
-
-		// DEVNOTE: It seems sometimes the message is just edited to remove data instead of actually retracting it
```
Comment (trek-boldly-go) [inline portal.go:2150]: I'd like to leave this dev note here for future reference.

**Review by trek-boldly-go — CHANGES_REQUESTED** [review]

File: portal.go (hunk 2 above)
Comment (cameronaaron) [reply portal.go:2149]: oops thanks

File: portal.go (hunk 3 above)
Comment (cameronaaron) [reply portal.go:2150]: yes

**Review by trek-boldly-go — APPROVED** [review]

**Pull request merged**